decide/voting/tests.py

```diff
@@ -238,6 +239,32 @@ def test_create_voting_from_api(self):
         response = self.client.post("/voting/", data, format="json")
         self.assertEqual(response.status_code, 201)
 
+    def test_create_boolean_voting_from_api(self):
+        data = {"name": "Example boolean"}
+        response = self.client.post("/voting/", data, format="json")
+        self.assertEqual(response.status_code, 401)
+
+        # login with user no admin
+        self.login(user="noadmin")
+        response = mods.post("voting", params=data, response=True)
+        self.assertEqual(response.status_code, 403)
+
+        # login with user admin
+        self.login()
+        response = mods.post("voting", params=data, response=True)
+        self.assertEqual(response.status_code, 400)
+
+        data = {
+            "name": "Example boolean",
+            "desc": "Description example boolean",
+            "question_type": "B",
+            "question": "I want a ",
+            "question_opt": ["cat", "dog"],
```

Comment:
Why are there questions and question options associated for a boolean object? If this is passing there is already something inherently wrong with the feature, this should be rejected, not allowed. When I create a boolean voting I want an automatic creation of the question options, which should be yes or no. If you are trying to see weather this gets ignored or not, you should check after the creation of the voting.

Reply:
I will check that part

Reply:
Now, it should be resolved. This test aims to check if it's possible to assign an example boolean question to a vote and consequently be voted.